vms/platformvm/warp/validator.go

```diff
@@ -53,7 +51,8 @@ func GetCanonicalValidatorSet(
 	}
 
 	var (
-		vdrs        = make(map[string]*Validator, len(vdrSet))
```

Comment:
line 36 here: yet another way to sort validators here. I wonder if we should let GetValidatorsSet do the "sort and filter unique" validators once and for all.